setup.py

```diff
@@ -29,10 +29,8 @@
 #: Packages requires for different environments
 EXTRA_PACKAGES = {
     ':sys_platform == "darwin"': [
-        'pyobjc-framework-Quartz >= 7.0, < 8.0',
-        'pyobjc-framework-Cocoa >= 7.0, < 8.0',
-        'pyobjc-core >= 7.0, < 8.0',
```

Comment:
The two packages (ApplicationServices, Quartz) might depend on core, and it would be good to pin transitive objc dependencies too. Would be good to at least check that.